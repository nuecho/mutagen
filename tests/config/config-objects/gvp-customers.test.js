importDependencies("gvp-customers-dependencies-config.json");
cfgObjectTests("gvpCustomers", "gvp-customers-config.json");