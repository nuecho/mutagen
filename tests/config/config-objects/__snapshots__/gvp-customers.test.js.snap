// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[mutagen gvpCustomers config import-export] should create the gvpCustomers: the gvpCustomers have been created 1`] = `
Object {
  "stderr": "",
  "stdout": "+ gvpcustomer => cfgObjectTest
Completed. 1 object(s) imported.
",
}
`;

exports[`[mutagen gvpCustomers config import-export] should update the gvpCustomers: the gvpCustomers have been updated 1`] = `
Object {
  "stderr": "",
  "stdout": "~ gvpcustomer => cfgObjectTest
Completed. 1 object(s) imported.
",
}
`;
